lib/rules/lowercase_description.js

```diff
@@ -0,0 +1,94 @@
+const { isEmpty: _isEmpty, has: _has, keys: _keys } = require("lodash");
+const { format } = require("util");
+const Rule = require("./Rule");
+
+module.exports = class LowercaseDescription extends Rule {
+    static meta = {
+        ruleId: "lowercase_description",
+        message: "Expected lowercase description on keyword '%s'",
+        type: "off",
+        location: {},
+        hasFix: true,
+        // lintAfterFix: false,
+    };
+
+    static run(ast, config) {
+        if (_isEmpty(ast?.feature)) return [];
+
+        return new LowercaseDescription(ast, config).execute();
+    }
+
+    execute() {
+        this.checkDescription(this._ast);
+
+        return this.getProblems();
+    }
+
+    checkDescription(astObject) {
+        const keyword = _keys(astObject).shift();
+        if (["rule", "feature", "background", "scenario"].includes(keyword)) {
+            if (
+                typeof astObject[keyword].name === "string" &&
+                astObject[keyword].name.length !== 0
```

Comment:
.name should be a string in every cases, right?
```suggestion
                astObject[keyword].name.length !== 0
```

Reply:
oh yes